src/SFML/Graphics/Font.cpp

```diff
@@ -121,6 +121,13 @@ float sfFont_getKerning(const sfFont* font, sfUint32 first, sfUint32 second, uns
 }
 
 
+////////////////////////////////////////////////////////////
+float sfFont_getBoldKerning(const sfFont* font, sfUint32 first, sfUint32 second, unsigned int characterSize)
+{
+    CSFML_CALL_RETURN(font, getKerning(first, second, characterSize, sfFalse == sfTrue), 0);
```

Comment:
The comparison with `sfTrue` makes sense if you have a `bool` value, here it just seems confusing, plus it's actually wrong, because we need to pass in `true` and `sfFalse == sfTrue` always evaluates to `false.

```suggestion
    CSFML_CALL_RETURN(font, getKerning(first, second, characterSize, true), 0);
```

Reply:
c doesn't have true and false boolean values. I did it with 1. 

I just realized that today. I mixed that up in my head when I was writing it.